vllm/compilation/pass_manager.py

```diff
@@ -31,24 +33,29 @@ class PostGradPassManager(CustomGraphPass):
     """
 
     def __init__(self):
-        self.passes: List[InductorPass] = []
+        self.passes: List[VllmInductorPass] = []
 
     def __call__(self, graph: fx.Graph):
+        shape = get_pass_context().runtime_shape
         for pass_ in self.passes:
-            pass_(graph)
+            if pass_.is_applicable_for_shape(shape):
```

Comment:
If this pass is an `InductorPass` instead of a `VllmInductorPass` (when added via `CompilationConfig.inductor_config` from outside vLLM), this method will not exist. We should either check this here or move the method to `InductorPass`